tdms/src/fields/electric.cpp

```diff
@@ -5,8 +5,8 @@
 using namespace std;
 
 
-void ElectricField::add_to_angular_norm(double f, int n, int Nt, SimulationParameters &params) {
-  angular_norm += phasor_norm(f, n, params.omega_an, params.dt, Nt);
+void ElectricField::add_to_angular_norm(int n, int Nt, SimulationParameters &params) {
+  angular_norm += phasor_norm(ft, n, params.omega_an, params.dt, Nt);
```

Comment:
Is `ft` coming from another file? Perhaps it should have a more explicit name, because at the moment it's unclear to me what `ft` is

Reply:
there's a ` TODO: ..` in the header with for an explanation, because I have no idea 😞